code/modules/reagents/withdrawal/generic_addictions.dm

```diff
@@ -95,7 +95,7 @@
 
 /datum/addiction/maintenance_drugs/withdrawal_enters_stage_1(mob/living/carbon/affected_carbon)
 	. = ..()
-	affected_carbon.apply_status_effect(/datum/status_effect/grouped/screwy_hud/fake_healthy, type)
+//	affected_carbon.apply_status_effect(/datum/status_effect/grouped/screwy_hud/fake_healthy, type) // Monke disable, this has nothing to do with maintenance.
```

Comment:
this does still make sense as a drawback so I say keep it

Reply:
I don't see what not knowing your health has to do with maintenance, greytiding or darkness. If I had to suggest an alternative drawback, I'd say something like kleptomania would fit better.